src/main/scala/PSO/PSOFactory.scala

```diff
@@ -6,7 +6,9 @@ import scala.concurrent.ExecutionContext.Implicits.global
 import akka.actor._
 
 class PSOFactory(config: PSOConfiguration) extends Executor {
```

Comment:
I might want to change the name from PSOFactory to PSOExecutor or something...because it's not a factory pattern anymore